### PR TITLE
CHANGELOG-1.12.1

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
+CHANGES IN VERSION 1.12.1 (FROM 1.12.0)
 ===================================
 
-#### General
-* Added `defaultImageTemplateScale` to arrange image comparison logic [#307](https://github.com/appium/appium-base-driver/pull/307). Read [doc](docs/en/advanced-concepts/image-elements.md) for more details.
+Appium 1.12.1 is a patch release for support Xcode 10.2 (iOS 12.2)
 
 #### iOS
-- Fix `useXctestrunFile` capability for newer Xcode [#903](https://github.com/appium/appium-xcuitest-driver/pull/903)
+- Fix Appium Remote Debugger to be compatible with Xcode 10.2 (addresses [issue #12239](https://github.com/appium/appium/issues/12239)) [#118](https://github.com/appium/appium-remote-debugger/pull/118)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: I'd rather say "Updated"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+- Fix problem with version comparison [#902](https://github.com/appium/appium-xcuitest-driver/pull/902)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: There was no problem. Just made it more reliable

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
+- Fix `screenshotQuality` cap not being forwarded to WebDriverAgent [#907](https://github.com/appium/appium-xcuitest-driver/pull/907)
+- Fix xctestrun file detection when `useXctestrunFile` is true [#903](https://github.com/appium/appium-xcuitest-driver/pull/903)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: I'd also include https://github.com/appium/appium-xcuitest-driver/pull/909 fix by @KazuCocoa

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,11 +1,13 @@
-DEVELOPING
+CHANGES IN VERSION 1.12.1 (FROM 1.12.0)
 ===================================
 
-#### General
-* Added `defaultImageTemplateScale` to arrange image comparison logic [#307](https://github.com/appium/appium-base-driver/pull/307). Read [doc](docs/en/advanced-concepts/image-elements.md) for more details.
```
Comment (KazuCocoa) [inline CHANGELOG.md]: can we bump https://github.com/appium/appium-base-driver/commits/master ?
(It includes only one fix for _find by image_ which has been fixed partially in 1.12.0)

Comment (dpgraham) [reply CHANGELOG.md]: I forgot about that. Do you mind if we do a patch release for it at a later date (near future)? We're testing the rc right now so we can support iOS 12.2 and I'd rather not introduce another dependency.

Comment (KazuCocoa) [reply CHANGELOG.md]: Okay. I don't mind that way. Thanks

**Review by KazuCocoa — APPROVED** [review]

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,11 +1,13 @@
...
+Appium 1.12.1 is a patch release that addresses WebView issues caused by Xcode 10.2 (iOS 12.2)
 
 #### iOS
-- Fix `useXctestrunFile` capability for newer Xcode [#903](https://github.com/appium/appium-xcuitest-driver/pull/903)
+- Updated Appium Remote Debugger so that Safari webview tests are compatible with Xcode 10.2 (addresses [issue #12239](https://github.com/appium/appium/issues/12239)) [#118](https://github.com/appium/appium-remote-debugger/pull/118)
+- Fix `screenshotQuality` cap not being forwarded to WebDriverAgent [#907](https://github.com/appium/appium-xcuitest-driver/pull/907)
+- Fix xctestrun file detection when `useXctestrunFile` is true [#903](https://github.com/appium/appium-xcuitest-driver/pull/903)
+- Fix problem with Appium not using cached WebDriverAgent [#909](https://github.com/appium/appium-xcuitest-driver/pull/909)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: A new cap has been added to android, which allows to set size or disable the remote cache: https://github.com/appium/appium-android-driver/commit/959e07d84dc9d09a11735872259ac87511d03b85

Comment (dpgraham) [reply CHANGELOG.md]: That isn't in this release. This one is just ios/xcui updates.

**Review by jlipps — APPROVED** [review]

**Pull request merged**